### PR TITLE
test(journeys): add data-api tests

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: test/journeys/server/data-api/space.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    id="ciscospark-widget">
+
+</div>
+<script>
+  window.loadBundle = function (bundleFile) {
+    var script = document.createElement('script');
+    script.src = bundleFile;
+    document.head.appendChild(script);
+    script.onload = function () {
+      console.info(`bundle loaded`);
```
Comment (bzang) [inline test/journeys/server/data-api/space.html]: Back ticks may be an issue in non chrome browsers. Should probably just keep everything in JS to single ticks.

File: test/journeys/specs/messageMeet/widget-message-meet/dataApi.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      `spark:memberships_read`,
+      `spark:memberships_write`,
+      `spark:messages_read`,
+      `spark:messages_write`,
+      `spark:teams_read`,
+      `spark:teams_write`,
+      `spark:team_memberships_read`,
+      `spark:team_memberships_write`,
+      `spark:kms`
+    ].join(` `);
```
Comment (bzang) [inline test/journeys/specs/messageMeet/widget-message-meet/dataApi.js]: Is this ready to be moved to spark:all yet?

File: test/journeys/server/data-api/space.html (hunk 1 above)
Comment (adamweeks) [inline test/journeys/server/data-api/space.html]: Yea, good call.

File: test/journeys/specs/messageMeet/widget-message-meet/dataApi.js (hunk 2 above)
Comment (adamweeks) [inline test/journeys/specs/messageMeet/widget-message-meet/dataApi.js]: We may still need `webexsquare.get_conversation` for some things, but I think `spark:all` should handle it. I'll try it out.

**Pull request merged**